packages/go/ein/ad.go

```diff
@@ -581,40 +583,50 @@ func ParseDCRegistryData(computer Computer) IngestibleNode {
 	var ()
 	propMap := make(map[string]any)
 
-	if computer.DCRegistryData.CertificateMappingMethods.Collected && computer.DCRegistryData.CertificateMappingMethods.Value >= 0 {
+	if computer.DCRegistryData.CertificateMappingMethods.Collected {
 		propMap[ad.CertificateMappingMethodsRaw.String()] = computer.DCRegistryData.CertificateMappingMethods.Value
 
-		var prettyMappings []string
+		if computer.DCRegistryData.CertificateMappingMethods.Value == -1 {
+			propMap[ad.CertificateMappingMethods.String()] = RegValNotExisting
```

Comment:
I'm not sure if this really matters, but in this case the value for the `CertificateMappingMethods` property will be a `string`, but in the case on L611 it will be an array. This might be annoying for API consumers?

Maybe we could remove the `>= 0` check on L592 and add a new `if` statement for this case in that block?